lib/data/demos.dart

```diff
@@ -859,13 +960,15 @@ List<GalleryDemo> referenceDemos(BuildContext context) {
           code: CodeSegments.transformationsDemo,
         ),
       ],
+      category: GalleryDemoCategory.other,
     ),
   ];
 }
 
 Map<String, GalleryDemo> slugToDemo(BuildContext context) {
+  final localizations = GalleryLocalizations.of(context);
   return LinkedHashMap<String, GalleryDemo>.fromIterable(
-    materialDemos(context) + cupertinoDemos(context) + referenceDemos(context),
+    allGalleryDemos(localizations),
     key: (dynamic demo) => demo.slug as String,
```

Comment:
@guidezpl I noticed in passing that this change looks like it adds a bunch of entries to the resulting map all with a `null` key.  That is, `allGalleryDemos` adds in the list of GalleryDemo entries produced from `studies`, each of which has `null` as the value of `demo.slug`.  The resulting map will have only a single entry for all of the `studies` entries, using `null` as the key.  Is this intentional?  Or have I somehow missed something here?

Reply:
Studies were implemented first, and then demos added the concept of slugs, so I think that's why it's the way it is. AFAIK studies don't actually use `slugToDemo`